doc/cs_registers.rst

```diff
@@ -76,7 +80,7 @@ Reset Value: ``0x0000_1800``
 +-------+-----+---------------------------------------------------------------------------------+
 
 When an exception is encountered, MPIE will be set to MIE.
-When the ``mret`` instruction is executed, the value of MPIE will be stored back to IE.
+When the MRET instruction is executed, the value of MPIE will be stored back to IE.
```

Comment:
Should that read `stored back to MIE`?

Beyond that, MIE vs mie is confusing in the RISC-V spec: MIE is a field in mstatus, while mie is a register. To avoid confusion, can we use mstatus.MPIE and mstatus.MIE here (I never understood why the spec uses all-caps for fields, but lowercase for registers, but hey ...)

Reply:
Oh, that's a good one. It should be MIE indeed. I fixed it also in another place and switched to the csr_num.FIELD_NAME syntax.